build/resources/main/application-dev.yml

```
@@ -0,0 +1,38 @@
+# 개발환경 Swagger 설정
+swagger:
+  server:
+    url: http://localhost:8080
+    description: 개발 서버
+
+# 개발환경 로깅
+logging:
+  level:
+    org.hibernate.SQL: debug
+    org.hibernate.type.descriptor.sql.BasicBinder: trace
+    com.example.login: debug
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)